swagger.yaml

```diff
@@ -32,6 +32,9 @@ paths:
           description: OK
           schema:
             $ref: "#/definitions/Tutorial"
+        "404":
```

Comment:
We also need to provide definition for the error response which will be common for all the APIs.

For example:
```
"404":
  description: "Tutorial not found"
  schema:
    $ref: "#/definitions/404Response"
```

Same for the 500 status code.